rails_translation_manager.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = ""
   spec.license       = "MIT"
 
-  spec.required_ruby_version = ">= 3.1"
+  spec.required_ruby_version = ">= 3.2"
```

Comment:
I've increased the required ruby version for the apps using this gem to 3.2. I didn't make it 3.3, because some apps do not use 3.3 yet.